internal/deploy/remote.go

```diff
@@ -60,13 +62,23 @@ func (c *remoteDeploymentManifest) ExecIn(ctx context.Context, profile ServiceRe
 
 // Inspect inspects a service
 func (c *remoteDeploymentManifest) Inspect(ctx context.Context, service ServiceRequest) (*ServiceManifest, error) {
+	var hostname string
 	// TODO: convert to a platform agnostic command structure
-	hostname, _ := shell.Execute(ctx, ".", "powershell.exe", "hostname")
+	if runtime.GOOS == "windows" {
```

Comment:
For the future, we may need to consume a library like GoSysinfo (https://github.com/elastic/go-sysinfo/blob/2b7239d64d62e2e26f626a71b4730dd78086d3f8/types/host.go#L67)

Reply:
Right now working with https://github.com/shirou/gopsutil